### PR TITLE
[devtools-blog] New post and add contributor: Migrating to web components

### PR DESCRIPTION
**Pull request by chybie**
**Target Live Date:** 2020-12-10

- [x] This has been reviewed and approved by (@chybie )
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

### CODE REVIEW

**Comment by chybie** [comment]
cc @jackfranklin

**Review by mathiasbynens — APPROVED** [review]
Rubberstamp LGTM

File: src/content/en/updates/2020/12/migrating-to-web-components.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+description: How we migrate Chrome DevTools to web components. The Chrome DevTools engineering blog - by the developers who build the DevTools.
+
+{# wf_updated_on: 2020-12-08 #}
+{# wf_published_on: 2020-12-08 #}
+{# wf_tags: devtools-blog #}
+{# wf_featured_image: /web/updates/images/2020/12/devtools-blog.png #}
+{# wf_featured_snippet: Web components is a great fit for building new UI elements in DevTools. To help with the transition, we created a guide to building UI elements in DevTools to share with the wider DevTools team. #}
+{# wf_blink_components: N/A #}
+
+# DevTools architecture refresh: Migrating to web components {: .page-title }
```
Comment (mathiasbynens) [inline src/content/en/updates/2020/12/migrating-to-web-components.md:12]:
```suggestion
# DevTools architecture refresh: migrating to web components {: .page-title }
```

File: src/content/en/updates/_shared/discover-devtools-blog.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 Below is a list of everything that's been covered in the *Chrome DevTools Engineering Blog* series.
 
+* [DevTools architecture refresh: Migrating to web components](/web/updates/2020/12/migrating-to-web-components)
```
Comment (mathiasbynens) [inline src/content/en/updates/_shared/discover-devtools-blog.md:5]:
```suggestion
* [DevTools architecture refresh: migrating to web components](/web/updates/2020/12/migrating-to-web-components)
```

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**